commands/argument.go

```diff
@@ -37,6 +38,16 @@ func FileArg(name string, required, variadic bool, description string) Argument
 	}
 }
 
+func StreamArg(name string, required, variadic bool, description string) Argument {
```

Comment:
maybe comment what `StreamArg` is used for? Im not sure myself atm. 

(i know the entire lib isnt well commented yet... perhaps we should lint the entire codebase)